docs/FeatureManagement/Samples/TargetingFilter.sample.json

```diff
@@ -0,0 +1,60 @@
+{
+    "feature_management": {
+        "feature_flags": [
+            {
+                "id": "ComplexTargeting",
+                "description": "A feature flag using a targeting filter, that will return true for Alice, Stage1, and 50% of Stage2, and false for Dave and Stage3. The default rollout percentage is 25%.",
+                "enabled": true,
+                "conditions": {
+                    "client_filters": [
+                        {
+                            "name": "Microsoft.Targeting",
+                            "parameters": {
+                                "Audience": {
+                                    "Users": [
+                                        "Alice"
+                                    ],
+                                    "Groups": [
+                                        {
+                                            "Name": "Stage1",
+                                            "RolloutPercentage": 100
+                                        },
+                                        {
+                                            "Name": "Stage2",
+                                            "RolloutPercentage": 50
+                                        }
+                                    ],
+                                    "DefaultRolloutPercentage": 25,
+                                    "Exclusion": {
+                                        "Users": ["Dave"],
+                                        "Groups": ["Stage3"]
+                                    }
+                                }
+                            }
+                        }
+                    ]
+                }
+            },
+            {
+                "id": "RolloutPercentageUpdate",
+                "description": "A feature flag using a targeting filter, that will return true 61% of the time. Changing to 62% makes the user Brittney true.",
```

Comment:
Since we have talked about Brittney will hit the 62%, should we roughly explain the percentage targeting algorithm somewhere?

Reply:
Why? These are validation tests for our feature management libraries. Do we expect customer to implement there own versions of the libraries that matches our own?

Reply:
My concern is that our customer will also look these samples. This is a blackbox for them. They may feel confused why Brittney hits 62%. But I am now kind of agree with you. This is validation test. People are not expected to learn feature management solely from these samples.